### DISCUSSION TITLE
Looping to Create Row

### DISCUSSION THREAD

**Discussion by smurraystrata** [Budibase/budibase#9975] [Help] [answered]
Hello!

I have an external data connector which provides 50 something rows, then I use a create row action with looping to add the results to a table. This has been working flawlessly for months now, until I updated to the latest version (2.4.8 running on docker compose). Now when I have the looping in place and set to steps.1.rows (as shown in the training video) with an input type of array it only loops once and fails to pass information to the fields (using loop.currentItem.differnetitems). I looked in the suggested bindings for the looping and saw steps.1.response. This causes an error (Error testing automation) to pop up. When I remove the looping and use bindings from step 1 it works (though only for the first Any suggestions what I can do to fix this?

The data from the External Data Connector has not changed. These screenshots are from a test copy of the automation going to a test table to prevent flooding the original table with the wrong data, but I get the same results on the original automation.

![Screenshot 2023-03-09 174139](https://user-images.githubusercontent.com/120056389/224193947-7cf6383c-beb9-4610-87d9-4f68888b1490.png)

![Screenshot 2023-03-09 174216](https://user-images.githubusercontent.com/120056389/224193959-6140fe75-5f09-4ffa-a5c7-e05ff7188e61.png)

Thank you all for the help and great tool!

**Comment by melohagan** [comment, marked as answer]
Hey @jedman20 

We believe this is a bug that will be fixed when the following is merged in: https://github.com/Budibase/budibase/pull/9984

Thank you for bringing this to our attention!

**Reply by smurraystrata** [reply]
I have updated to the latest release and it is now working. Thank you again!
